drivers/hmis/app/models/hmis/form/definition_item_filter.rb

```diff
@@ -32,7 +32,10 @@ def eval_items(items)
     # return items if Rails.env.development?
 
     items.filter do |item|
-      if eval_rule(item['rule'])
+      true_for_hud_rule = eval_rule(item['rule'])            # HUD system rules that cannot be overridden
+      true_for_custom_rule = eval_rule(item['custom_rule'])  # Custom rules that can be managed by admins
+
+      if true_for_hud_rule || true_for_custom_rule
```

Comment:
micro performance optimization would be to use lazy eval here so that we only need to run the first eval_rule if it's true
```suggestion
      if eval_rule(item['rule'])  || eval_rule(item['custom_rule'])
```